src/main/java/org/aludratest/config/AludraTestConfig.java

```diff
@@ -47,6 +48,9 @@
     /** Configuration property name. */
     public static final String STOP_ON_INTERACTION_PROP = "stop.testcase.on.interaction.exception";
 
+    /** Configuration property name. */
+    public static final String LOGGING_DISABLED_PROP = "aludratest.logging.disabled";
```

Comment:
Please remove aludratest. prefix. It is not required, as this property is located in aludratest.properties, so unique identifier is given.